swarm/pss/pssapi.go

```diff
@@ -90,48 +103,77 @@ func (pssapi *API) GetPublicKey() []byte {
 }
 
 // Set Public key to associate with a particular Pss peer
-func (pssapi *API) SetPeerPublicKey(addr []byte, topic whisper.TopicType, pubkey []byte) error {
-	var potaddr pot.Address
-	copy(potaddr[:], addr)
-	pssapi.Pss.SetPeerPublicKey(potaddr, topic, crypto.ToECDSAPub(pubkey))
+func (pssapi *API) SetPeerPublicKey(pubkey []byte, topic whisper.TopicType, addr PssAddress) error {
+	pssapi.Pss.SetPeerPublicKey(crypto.ToECDSAPub(pubkey), topic, addr)
 	return nil
 }
 
+// Set Symmetric key for communication with a particular Pss peer
+//
+// if the addtocache parameter is set to false, pss will add this key to the collection of keys used to decrypt incoming messages.
+//
+// returns the string id auto generated by whisper. Use this to retrieve the key later with GetSymKey
+func (pssapi *API) SetPeerSymmetricKey(symkey []byte, topic whisper.TopicType, addr PssAddress, expires time.Duration, addtocache bool) (string, error) {
```

Comment:
`SetSymmetricKey` is already defined on the API since Pss is an embedded type. 
Could we not just use that?